### PR TITLE
Add text for CPU, memory, and storage guideline.

### LINKED ISSUE #10: Add CPU and storage management to fair use
I will do this myself.

### PR DESCRIPTION
**Pull request by pwrightkcl**
Feel free to move these around. I just wanted to provide the basic guidelines. Happy to leave organisation / formatting to whoever is owning this page.

Closes #10

### CODE REVIEW

File: DGX_fair_use_guidelines.md [hunk 1]
```diff
@@ -60,3 +60,70 @@ right:
...
+Run:ai is designed to allocated GPU resources fairly, but CPU resources are not managed in the same way. It is up to 
+users to submit jobs with appropriate limits on CPU and CPU memory resources. A single job can use up the available 
+resources on a node and prevent all the other jobs from running, even if the GPU resources are available.
+
+Use the [--cpu-limit](https://docs.run.ai/v2.16/Researcher/cli-reference/runai-submit/#-cpu-limit-double) 
+and [--memory-limit](https://docs.run.ai/v2.16/Researcher/cli-reference/runai-submit/#-memory-limit-string) flags in 
+your `runai submit` command to set these limits. As a rule of thumb, divide the CPU and memory resources of the node by 
+the number of GPUs and set your limits according to how many GPUs you requested.
+
+Occasionally, you may need to run a CPU-only job on the server. Keep an eye on the general usage on the cluster and on
```
Comment (mhubii) [inline DGX_fair_use_guidelines.md:74]: might be helpful to explain how target nodes can be specified here, which isn't documented somewhere

File: DGX_fair_use_guidelines.md [hunk 2]
```diff
@@ -60,3 +60,70 @@ right:
...
+  * If you "get hit by a bus", having your work organised with the intent of sharing it will make it easier for your
+colleagues to pick up where you left off.
+* Consider archiving large datasets.
+  * If you are no longer using a specific dataset, consider moving it off `/nfs`.
+
+## Fair use of disk I/O
+
+The entire cluster runs on a single storage system. If too many processes are trying to read and write from the disk at 
+once then the system will slow down for everyone. Try to optimise your code to avoid repeated reads from the same file. 
+Some modules / data loaders will allow you to explicitly cache data in memory.
```
Comment (mhubii) [inline DGX_fair_use_guidelines.md:112]: this can cause issues with cpu memory (like the MONAI cached dataset)

File: DGX_fair_use_guidelines.md [hunk 3]
```diff
@@ -60,3 +60,70 @@ right:
...
+your `runai submit` command to set these limits. As a rule of thumb, divide the CPU and memory resources of the node by 
+the number of GPUs and set your limits according to how many GPUs you requested.
+
+Occasionally, you may need to run a CPU-only job on the server. Keep an eye on the general usage on the cluster and on 
+the node you are using and adjust your limits. It may help to break your processing into separate jobs; that way you can 
+submit as many as the node can handle and wait until they complete before submitting more.
+
+### Memory leaks
+
+In the Run:ai interface, monitor the CPU memory usage of your job. These appear in the "Metrics" charts shown above,
```
Comment (mhubii) [inline DGX_fair_use_guidelines.md:80]: feel like this section can be shortened much

File: DGX_fair_use_guidelines.md [hunk 4]
```diff
@@ -60,3 +60,70 @@ right:
...
+
+## Fair use of disk I/O
+
+The entire cluster runs on a single storage system. If too many processes are trying to read and write from the disk at 
+once then the system will slow down for everyone. Try to optimise your code to avoid repeated reads from the same file. 
+Some modules / data loaders will allow you to explicitly cache data in memory.
+
+This is where we enter slightly controversial territory. Some sysadmins have observed that VScode and possibly Pycharm 
+spawn hundreds of background processes, some of which are constantly checking the filesystem for changes. It has been 
+reported that adding a script to delete `.vscode` processes improved IO performances. Moreover before this change was
```
Comment (mhubii) [inline DGX_fair_use_guidelines.md:116]: to this day I don't see any evidence for this claim

**Comment by mhubii** [comment]
thanks @pwrightkcl for this contribution. Happy to merge, but please consider the review. Also happy to give you admin rights to this repo if needed

File: DGX_fair_use_guidelines.md (hunk 4 above)
Comment (pwrightkcl) [reply DGX_fair_use_guidelines.md:116]: That's why I said it's controversial. Did you ever discuss it with Eric? He's the one that investigated this rigorously on Jade. If you want to tone down the wording even further, I'm happy to do that, just want the info out there so people can choose.

File: DGX_fair_use_guidelines.md (hunk 1 above)
Comment (pwrightkcl) [reply DGX_fair_use_guidelines.md:74]: Added tip

File: DGX_fair_use_guidelines.md (hunk 2 above)
Comment (pwrightkcl) [reply DGX_fair_use_guidelines.md]: Added caveat

File: DGX_fair_use_guidelines.md (hunk 3 above)
Comment (pwrightkcl) [reply DGX_fair_use_guidelines.md]: Shortened

**Comment by pwrightkcl** [comment]
> thanks @pwrightkcl for this contribution. Happy to merge, but please consider the review. Also happy to give you admin rights to this repo if needed

Thanks for the review. I've edited all but the last section because I'm not sure what you had in mind (for the part about vscode). I think at least the first one - don't leave projects open when you're not working on them - should be easy and uncontroversial. The others are all "consider x" so left to the reader. Happy for you to edit if you want that clearer.

File: DGX_fair_use_guidelines.md (hunk 4 above)
Comment (mhubii) [reply DGX_fair_use_guidelines.md:118]: I think its fine

**Pull request merged**